core/flyout_base.ts

```diff
@@ -1377,13 +932,37 @@ export abstract class Flyout
     // No 'reason' provided since events are disabled.
     block.moveTo(new Coordinate(finalOffset.x, finalOffset.y));
   }
+
+  /**
+   * Returns the inflater responsible for constructing items of the given type.
+   *
+   * @param type The type of flyout content item to provide an inflater for.
+   * @returns An inflater object for the given type, or undefined if no inflater
+   *     is registered for that type.
+   */
+  protected getInflaterForType(type: string): IFlyoutInflater | undefined {
+    if (this.inflaters.has(type)) {
+      return this.inflaters.get(type);
+    }
+
+    const InflaterClass = registry.getClass(
+      registry.Type.FLYOUT_INFLATER,
+      type,
+    );
+    if (InflaterClass) {
+      const inflater = new InflaterClass();
+      this.inflaters.set(type, inflater);
+      return inflater;
+    }
+
+    return undefined;
```

Comment:
One last nit: why `undefined` instead of `null` here?

Reply:
Because that was `Map`'s not found return type, but updated for consistency.